src/main/java/com/iexec/worker/compute/ComputeManagerService.java

```diff
@@ -101,11 +85,26 @@ public boolean downloadApp(TaskDescription taskDescription) {
                 .pullImage(taskDescription.getAppUri(), Duration.of(pullTimeout, ChronoUnit.SECONDS));
     }
 
+    /**
+     * Examples of timeout depending on category:
+     * <ul>
+     *     <li>XS:   50 * 0.02 = 7 minute</li>
```

Comment:
1. 0.02 is not update to date
2. I think a new reader might be lost between comments in minutes and implementation in seconds, maybe we should stick to one unit and express the other one in parenthesis for human readability (just an idea)
3. I would also add the formula in the comment

Reply:
Thanks for the idea, see https://github.com/iExecBlockchainComputing/iexec-worker/pull/442/commits/2c95f49a424dc7fdb2164edf6e86b7244898257b.

Reply:
Thanks